cli/src/main.rs

```diff
@@ -463,6 +483,43 @@ async fn run_cache(command: &CacheCommand) -> Result<(), CliError> {
     Ok(())
 }
 
+async fn run_init(args: &InitArgs) -> Result<(), CliError> {
+    match args.language.to_lowercase().trim() {
```

Comment:
I think the way to go here may not be to actually have a `match` here that includes hard-coded URLs, it would probably be best to have a map of sorts, like we do for `package-registry`. I am fully aware that for three projects this works fine, and it seems unnecessary to complicate things further in this early stage, but can we go half-way and like `include_str!` a file containing JSON mappings to things? Like if we have a file `templates.json`:
```json
{
    "https://github.com/modmark-org/rust_template": ["rust", "rs"],
    "https://github.com/modmark-org/cpp_template": ["c++", "cpp"],
    "https://github.com/modmark-org/c_template": ["c"]
}
```
, then `let mappings: HashMap<String, Vec<String>> = serde_json::from_json(include_str!("templates.json"));` statically once, and do searches like `mappings.iter().find(|(k, v)| v.contains(args.language.to_lowercase().trim()).then_some(k))`